### PR TITLE
Edited the ovirt-permissions role.

### PR DESCRIPTION
**Pull request by bgraveno**
Hi @machacekondra,

Could you please review and merge my changes?

Kind regards,

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by machacekondra** [comment]
ci test please

**Comment by bgraveno** [comment]
Hi @machacekondra, 

I've fixed that issue.

Thanks,

File: roles/ovirt-permissions/README.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ Requirements
...
 |---------------|----------------|----------------------------|
-| state         | present        | State of the permission    |
-| user_name     | UNDEF          | User name of user to manage permission |
-| group_name    | UNDEF          | Name of group to manage permission |
-| authz_name    | UNDEF          | Name of the authorization provider of the group or user |
-| role          | UNDEF          | Role to be assigned to user or group |
-| object_type   | UNDEF          | The object type which should be used to assign permission |
-| object_name   | UNDEF          | Name of the object where permission should be assigned |
+| state         | present        | Specifies whether the state of the permission is `present` or `absent`.    |
+| user_name     | UNDEF          | The user to manage the permission. |
```
Comment (mwperina) [inline roles/ovirt-permissions/README.md]: Shouldn't it be changed to 'The user to manage the permission for.'?

File: roles/ovirt-permissions/README.md [hunk 2]
```diff
@@ -12,17 +12,17 @@ Requirements
...
+| group_name    | UNDEF          | Name of group to manage the permission. |
```
Comment (mwperina) [inline roles/ovirt-permissions/README.md]: Shouldn't it be changed to 'The group to manage the permission for.'?

File: roles/ovirt-permissions/README.md [hunk 3]
```diff
@@ -12,17 +12,17 @@ Requirements
...
+| authz_name    | UNDEF          | Name of the authorization provider of the group or user. |
+| role          | UNDEF          | The role to be assigned to the user or group. |
+| object_type   | UNDEF          | The object type which should be used to assign the permission. Possible object types are: <br> <ul><li>data_center</li><li>cluster</li><li>host</li><li>storage_domain</li><li>network</li><li>disk</li><li>vm</li><li>vm_pool</li><li>template</li><li>cpu_profile</li><li>disk_profile</li><li>vnic_profile</li><li>system</li></ul> |
+| object_name   | UNDEF          | Name of the object where the permission should be assigned. |
```
Comment (mwperina) [inline roles/ovirt-permissions/README.md]: Again do we want to allow HTML tags inside documentation?

**Review by mwperina — CHANGES_REQUESTED** [review]

**Comment by machacekondra** [comment]
@bgraveno Hi, can you please answer Martin's questions, I am unsure what's gramaticaly correct.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

File: roles/ovirt-permissions/README.md (hunk 2 above)
Comment (bgraveno) [reply roles/ovirt-permissions/README.md]: I'll make this update. I wasn't sure if we meant, "The user to manage the permission.", or "The user to manage the permission for."

**Comment by bgraveno** [comment]
Made the suggested changes.

**Pull request merged**